src/nlsolve/utils.jl

```diff
@@ -108,16 +108,19 @@ DiffEqBase.@def oopnlsolve begin
       end
       W = WOperator(f.mass_matrix, dt, J, false)
     else
-      if DiffEqBase.has_jac(f)
-        J = f.jac(uprev, p, t)
-      else
-        if alg_autodiff(alg)
-          J = jacobian_autodiff(uf, uprev)
-        else
-          J = jacobian_finitediff(uf, uprev, alg.diff_type)
-        end
-      end
-      W = J isa Number ? J : lu(J; check=false)
+      #if DiffEqBase.has_jac(f)
+      #  J = f.jac(uprev, p, t)
+      #else
+      #  if alg_autodiff(alg)
+      #    J = jacobian_autodiff(uf, uprev)
+      #  else
+      #    J = jacobian_finitediff(uf, uprev, alg.diff_type)
+      #  end
+      #end
+      #W = J isa Number ? J : lu(J; check=false)
+      W = u isa Number ? u : LU{LinearAlgebra.lutype(eltype(u))}(Matrix{uEltypeNoUnits}(undef, 0, 0),
```

Comment:
how will alternative linsolves and J types work here?

Reply:
We don't have the option to switch linsolve on OOP solvers, yet.

Reply:
oh yeah true. And there's not really a reason for it either.